### PR TITLE
Stripe payments v3

### PR DESCRIPTION
**Pull request by SJanvekar**
Check and lmk

### CODE REVIEW

File: lib/Main.dart [hunk 1]
```diff
@@ -128,7 +133,7 @@ class _MainPageState extends State<MainPage>
     controller.forward();
     getUserDetails();
     setState(() {
-      print('main set');
+      print(userInstance.userType);
```
Comment (wxianBalance) [inline lib/Main.dart:136]: Remove if not needed

File: lib/Requests/StripeRequests.dart [hunk 2]
```diff
@@ -39,4 +39,25 @@ class StripeRequests {
       print("Stripe Account Retrival Error: ${e}");
     }
   }
+
+  //Create payment intent
+  newPaymentIntent(
+    customerID,
```
Comment (wxianBalance) [inline lib/Requests/StripeRequests.dart:45]: Put the data type for these params, do it for the other ones in this files too

File: lib/sharedWidgets/classes/ClassListHome.dart [hunk 3]
```diff
@@ -20,18 +20,18 @@ class _ClassListHomeState extends State<ClassListHome> {
...
-              physics: const NeverScrollableScrollPhysics(),
-              scrollDirection: Axis.vertical,
-              itemCount: classList.length,
-              itemBuilder: (context, index) {
-                final classItem = classList[index];
-                return HomeClassItem(
-                  classItem: classItem,
-                );
-              })
+          // ListView.builder(
```
Comment (wxianBalance) [inline lib/sharedWidgets/classes/ClassListHome.dart:23]: Remove if not needed

File: lib/sharedWidgets/fitsySharedLogic/StripeLogic.dart [hunk 4]
```diff
@@ -110,4 +106,14 @@ class StripeLogic {
       }
     });
   }
+
+  // //Check if details are submitted
+  // makePayment(
+  //   User userInstance,
+  //   User trainerInstance,
```
Comment (wxianBalance) [inline lib/sharedWidgets/fitsySharedLogic/StripeLogic.dart:113]: Remove if not needed

File: lib/Main.dart (hunk 1 above)
Comment (SJanvekar) [reply lib/Main.dart:136]: Removed

File: lib/sharedWidgets/classes/ClassListHome.dart (hunk 3 above)
Comment (SJanvekar) [reply lib/sharedWidgets/classes/ClassListHome.dart:23]: File deleted

File: lib/sharedWidgets/fitsySharedLogic/StripeLogic.dart (hunk 4 above)
Comment (SJanvekar) [reply lib/sharedWidgets/fitsySharedLogic/StripeLogic.dart:113]: Removed

File: lib/screen/home/components/HomeClassCard.dart [hunk 5]
```diff
@@ -0,0 +1,481 @@
...
+                    fontSize: 14,
+                    fontWeight: FontWeight.w400)),
+          )
+        ],
+      ),
+    ),
+  );
+}
+
+//Unused Widgets
```
Comment (wxianBalance) [inline lib/screen/home/components/HomeClassCard.dart:434]: Remove if not needed

File: lib/Requests/StripeRequests.dart
Comment (wxianBalance) [inline lib/Requests/StripeRequests.dart:1]: Can you add a data type to the other functions in this class too?

**Review by wxianBalance — CHANGES_REQUESTED** [review]
Remember to pull main to merge my changes

**Review by wxianBalance — APPROVED** [review]

**Pull request merged**